plugins/bitrise/README.md

```diff
@@ -11,12 +11,6 @@ Welcome to the Bitrise plugin!
 $ yarn add @backstage/plugin-bitrise
 ```
 
-Then make sure to export the plugin in your app's [`plugins.ts`](https://github.com/backstage/backstage/blob/master/packages/app/src/plugins.ts) to enable the plugin:
-
-```js
-export { bitrisePlugin } from '@backstage/plugin-bitrise';
-```
-
```

Comment:
one more thing. mine adds some more changes

https://github.com/backstage/backstage/pull/5452/files

is something in there of interest to also address as part of this pr?

Reply:
Cherry-pick'd and resolved conflicts